ext/drutil/drutil.h

```diff
@@ -110,6 +110,24 @@ DR_EXPORT
  * to be passed in.
  * For single-instruction string loops, returns the size referenced
  * by each iteration.
+ *
+ * If the instruction is part of the xsave family of instructions, this
+ * returns an incomplete computation of the xsave instruction's written
+ * xsave area's size. Specifically, it
+ *
+ * - Ignores the user state mask components set in edx:eax, because it is
```

Comment:
"it" here is confusing: it has to refer to the "it" on line 116.  I think you mean to refer to the mask components, for which you cannot use "it".  s/it is a dynamic value/they are dynamic values/.

Reply:
Thanks, fixed.